src/DotNet/Writer/Metadata.cs

```diff
@@ -445,6 +445,7 @@ public abstract class Metadata : IReuseChunk, ISignatureWriterHelper, ITokenProv
 		RVA rva;
 		readonly MetadataOptions options;
 		ILogger logger;
+		internal readonly MetadataErrorContext errorContext;
```

Comment:
Does this need to be `internal`?

Reply:
MetadataErrorContext is internal type and I don't know whether child type like NormalMetadata will use errorContext to mark error source in the future. So I set errorContext to internal.

Reply:
`internal` can be added if needed in the future if that ever happens. It's better to not add `internal` if not needed.